docs/administration/configuration/hashicorp-consul.md

```diff
@@ -0,0 +1,87 @@
+# HashiCorp Consul (Enterprise)
+
+Rundeck Enterprise users have the option to use HashiCorp Consul as a method to edit Rundeck's configuration properties. By doing so, it allows users to manage the configuration for all of their services in one place. In terms of Rundeck, this feature is very beneficial because it allows users to edit configurations across all of their Rundeck instances. So, instead of having to change a configuration for each instance, you can change it once in Hashicorp and it will apply to all instances.  
+
+## Getting Started 
+
+To begin, we need to make sure that we have both Rundeck and HashiCorp Consul installed. The easiest way to get HashiCorp up and running is using their docker image. The docker image is named "consul." Consul can be found at localhost:8500, but this can also be changed from within Consul. 
```

Comment:
Let's leave out a portion of this and instead borrow from their docs or point to their docs on how to implement consul.  We can also add a folder to docker-zoo with a Consul for Config setup as an example.

Reply:
This comment wasn't addressed in latest updates.  Let's link to their install docs and tighten this up a bit.